dummy_core/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "screencrash_dummy_core",
+  "version": "1.0.0",
+  "description": "Dummy setup to be able to feed manual commands to components.",
+  "main": "index.js",
+  "scripts": {
```

Comment:
I think you can add a script "start": "electron ." or similar. Then you can start the component by running "npm start" instead of pointing to electron.exe.

("start" is handled in a special way so "npm start" works. Normally you have to do "npm run XX".)

Reply:
Aha, I commented on the wrong package.json. The other one already has it.